sale.py

```diff
@@ -217,7 +217,7 @@ def pos_find_sale_line_domain(self):
 
         return domain
 
-    def pos_add_product(self, product_ids, quantity):
+    def pos_add_product(self, product_ids, line):
```

Comment:
If `line` can have `quantity` and `unitprice` then `line` can also have `product_id` then method should be called `pos_add_sale_line`.

Preferred approach is to add `unitprice` as *param*